Packs/HYASInsight/Integrations/HYASInsight/README.md

```diff
@@ -673,9 +555,101 @@ Returns associated Domain's for the provided hash value.
 #### Human Readable Output
 
 >### HYAS HASH-DOMAIN records for md5 : 1d0a97c41afe5540edd0a8c1fb9a0f2d
->|Associated Domains|
->|---|
->| domain.es |
->| qwertasdfg.sinip.es |
->| butterfly.bigmoney.biz |
+>**No entries.**
```

Comment:
Would be better to provide an example with existing Human Readable Output entries.